src/main/java/bob/Task.java

```diff
@@ -0,0 +1,60 @@
+package bob;
+
+/**
+ * Represents Task object, something for the user to take note of
+ */
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for Task object with description
+     *
+     * @param description name or details of task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Retrieves status of completion of task
+     *
+     * @return "[X]" if task is done and "[ ]" if task is not
+     */
+    public String getStatusIcon() {
+        return (isDone? "[X]" : "[ ]");
+    }
+
+    /**
+     * Marks or Unmarks task depending on boolean input
+     *
+     * @param x boolean value indicating if task should be marked as completed or not
+     */
+    public void toMark(boolean x) {
```

Comment:
Perhaps you can rename this function to `mark` to make its purpose clearer?
```suggestion
    public void mark(boolean x) {
```

Reply:
Okay, thank you!